dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/export/TrackerEventsExportController.java

```diff
@@ -124,6 +125,48 @@
 
     }
 
+    @GetMapping( produces = { "application/csv", "application/csv+gzip", "text/csv" } )
```

Comment:
Could you extract some constants like we do for the APPLICATION_JSON_VALUE
Json is even here https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/http/MediaType.html the others unfortunately not

Reply:
Done